contrib/nixos/goshy.nix

```diff
@@ -0,0 +1,94 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.goshy;
+
+  goshy = pkgs.writers.writeBashBin "goshy" ''
```

Comment:
Would be neat to read the same goshy script as in `contrib/bash/goshy`. By doing so, we avoid duplicate code basis and inconsistency after changes.

Reply:
One could use [`makeWrapper`](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper) to default the configured options.

Reply:
I did, thanks for the hint! Resolved with 6b6b95c.